src/jit/gentree.cpp

```diff
@@ -7619,6 +7881,14 @@ GenTreePtr Compiler::gtCloneExpr(GenTree* tree,
                                                      tree->gtBoundsChk.gtThrowKind);
             break;
 
+        case GT_STORE_DYN_BLK:
+        case GT_DYN_BLK:
+            copy = new (this, oper) GenTreeDynBlk(gtCloneExpr(tree->gtDynBlk.Addr(), addFlags, varNum, varVal),
+                                                  gtCloneExpr(tree->gtDynBlk.gtDynamicSize, addFlags, varNum, varVal));
+            // ??? WHY IS THIS HERE?? THE ABOVE SHOULD HAVE SET IT CORRECTLY ???
+            // copy->gtDynBlk.gtOp1 = tree->gtDynBlk.gtOp1;
+            break;
```

Comment:
Another missed cleanup. I wasn't sure why I'd added it, so I commented it out and then failed to remove it.

---
In reply to: [77259059](https://github.com/dotnet/coreclr/pull/7027#discussion_r77259059) [](ancestors = 77259059)